app/models/katello/rhsm_fact_parser.rb

```diff
@@ -61,7 +61,7 @@ def operatingsystem
 
     def os_release_name(os_name)
       if os_name.match(::Operatingsystem::FAMILIES['Debian'])
-        facts['distribution.id']
+        facts['distribution.id']&.split(' ')&.first&.downcase
```

Comment:
split(' ') is the same as .split without any args

Reply:
agreed, looks nicer :+1: